watertap/property_models/activated_sludge/asm2d_properties.py

```diff
@@ -290,7 +290,7 @@ def build(self):
         self.temperature = pyo.Var(
             domain=pyo.NonNegativeReals,
             initialize=298.15,
-            bounds=(298.15, 323.15),
+            bounds=(298.14, 323.15),
```

Comment:
I suppose the lower bound on this can be something like 283.15K but not critical right now.

Reply:
Yes but would have to be careful as all the reaction parameters depend on temperature, so if someone wants to use the model at a lower temperature would have to rewrite those parameter values